refactor: serialization cart size check (#11865)
shopware/platform: changelog/_unreleased/2025-08-12-serialization-cart-size-check.md

---
title: Serialization cart size check
issue: #10737
---
# Core
* Added new method `CartException::serializeCartTooLarge`

> * Added `shopware.cart.serialization_max_size` config option to define the maximum size (in MB) of the serialized and compressed cart. No default value is set.

Reviewer: ```suggestion
* Added optional `shopware.cart.serialization_max_size` config option to define the maximum size (in MB) of the serialized and compressed cart.
```
